sklearn/neighbors/tests/test_nearest_centroid.py

```diff
@@ -3,17 +3,28 @@
 """
 import numpy as np
 import pytest
-from numpy.testing import assert_array_equal
+from numpy.testing import assert_array_almost_equal, assert_array_equal
```

Comment:
Can you change this import to import from `sklearn.utils._testing` instead of `numpy.testing.`